packages/gestalt/src/Typeahead.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable eslint-comments/disable-enable-pair */
```

Comment:
@simkessy could we remove these 2 eslint disable comments?

If it's still required, please use `eslint-disable-next-line` or `eslint-disable-line` instead to make it specific to a line.